include/wx/generic/grid.h

```diff
@@ -2267,6 +2296,12 @@ class WXDLLIMPEXP_CORE wxGrid : public wxScrolledCanvas
     // make the cell editable/readonly
     void SetReadOnly(int row, int col, bool isReadOnly = true);
 
+    // Return @true if overlay selection can be used (wxUSE_GRAPHICS_CONTEXT=1)
+    // and DisableOverlaySelection() hadn't been called.
+    static bool UsesOverlaySelection();
+
+    static void DisableOverlaySelection();
```

Comment:
Sorry, but why do these functions have to be global, i.e. class-static? I thought they would be per-grid, so that you could customize the selection of a particular grid without affecting anything else.

Maybe it would make sense to have some global `DisableOverlaySelectionByDefault()` that would set `gs_useOverlaySelection` to `false`, but I'd still add `m_useOverlaySelection`, that would be initialized with `gs_useOverlaySelection` but could be set to `false` using non-static `DisableOverlaySelection()` without affecting any other grids.

Reply:
Done without adding `DisableOverlaySelectionByDefault()`.